bulbs/videos/migrations/0002_auto_20160613_1116.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@mparent61 This file was generated when I generated my migrations. Do we want to add this?